core-blocks/image/index.js

```diff
@@ -47,6 +47,20 @@ const blockAttributes = {
 		source: 'attribute',
 		selector: 'figure > a',
 		attribute: 'href',
+		default: null,
```

Comment:
It was because I kept receiving this error when running tests. I wasn't exactly sure what to do so please feel free to suggest and I'll look into it!

<img width="603" alt="screen shot 2018-05-31 at 3 19 40 pm" src="https://user-images.githubusercontent.com/8163241/40806135-2d8a3394-64e6-11e8-83e0-bf83fb7f350f.png">